environment:
  lower: -5.0
  upper: 5.0
  num_of_obstacles: 12
  environment_state_dim: 20
  agent_state_dim: 4
  action_dim: 2
  agent_name: "pioneer_3dx"
  max_action: 1.0
  actions_low: [-1.0, -1.0]
  actions_high: [1.0, 1.0]

threshold_parameters:
  goal_threshold: 0.3
  collision_threshold: 0.35
  time_delta: 0.1
  inter_entity_distance: 1.5
  lidar_max_range: 10
